examples/CloudFunction/README.md

```diff
@@ -0,0 +1,121 @@
+# Terraform Stackdriver Aggregated Export to PubSub
+
+The Terraform module can be used to provision a cloudfunction which gets Gsuite Admin Logs and syncs them to stackdriver
+
+### APIs
+For the GSuite Exporter to work, the following APIs must be enabled in the project:
+- Identity and Access Management API: `iam.googleapis.com`
+- Admin SDK API: `admin.googleapis.com`
+- PubSub API: `pubsub.googleapis.com`
+- Cloud Functions API: `cloudfunctions.googleapis.com`
+- Cloud Scheduler API: `cloudscheduler.googleapis.com`
+- App Engine API: `appengine.googleapis.com`
+
+
+## Prerequisites 
+
+The following GSuite Admin APIs are currently supported:
+
+- `reports_v1` - [Reports API](https://developers.google.com/admin-sdk/reports/v1/get-start/getting-started)
+    - `admin` - [Admin activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-admin.html)
+    - `drive` - [Google Drive activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-drive.html)
+    - `login` - [Login activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-login.html)
+    - `mobile` - [Mobile activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-mobile.html)
+    - `token` - [Authorization Token activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-tokens.html)
+
+
+The following destinations are currently supported:
+- [Stackdriver Logging](https://cloud.google.com/logging/docs/)
+
```

Comment:
Lines 5 - 29: Please don't copy paste the gsuite-exporter documentation but link to the [gsuite-exporter docs](https://github.com/GoogleCloudPlatform/professional-services/tree/master/tools/gsuite-exporter) instead.